### PR TITLE
Re-ordering of DAMS elements

### PR DESCRIPTION
**Pull request by VivianChu**
References #632, https://github.com/ucsdlib/damsmanager/issues/368 

#### Local Checklist
- [ ] Tests written and passing locally?
- [x] Code style checked?
- [x] QA-ed locally?
- [x] Rebased with `master` branch?
- [ ] Configuration updated (if needed)?
- [ ] Documentation updated (if needed)?

#### What does this PR do?
Re-ordering of DAMS elements in the UI.

#### Where should a reviewer start?
It's deployed to qa

#### Manual testing steps?
For example
//rdc object
http://libraryqa.ucsd.edu/dc/object/bb04139816
http://libraryqa.ucsd.edu/dc/object/bb2018464v

//non-rdc object
http://libraryqa.ucsd.edu/dc/object/bb5157099g
http://libraryqa.ucsd.edu/dc/object/bb1574705v
http://libraryqa.ucsd.edu/dc/object/bb2083712b

@ucsdlib/developers - please review

### CODE REVIEW

**Comment by VivianChu** [comment]
Thanks @mcritchlow for reviewing.  @hjsyoo already reviewed on QA. https://github.com/ucsdlib/damspas/issues/632#issuecomment-542932437

**Comment by gamontoya** [comment]
@VivianChu This adjustment looks good.

**Comment by lsitu** [comment]
@VivianChu I looks good and I just have a little question. Maybe we also need a test to validate `cite this work` are displayed for `ldcp` collections while `preferred_citation_note` are displayed for `dlp` collections?

**Comment by hjsyoo** [comment]
@lsitu For clarification, Cite This Work should be the displayed label for all objects regardless of Unit/Repository. Unit should only affect the relative placement of Note:Preferred citation.
https://github.com/ucsdlib/damsmanager/issues/368#issuecomment-542347090

**Comment by hjsyoo** [comment]
@lsitu Sorry about the confusion. I should have updated the specs, and will do so now. The related ticket, https://github.com/ucsdlib/damsmanager/issues/368, has new specs as discussed at the last dams-wg meeting, so the name of the ticket is currently misleading. To summarize https://github.com/ucsdlib/damspas/issues/632 and https://github.com/ucsdlib/damsmanager/issues/368, we would like the DAMS elements to be re-ordered as indicated in https://docs.google.com/spreadsheets/d/1n7REAqpt2vh_BQWXFhTbAyVQY5kI7JPOuBAhg8WYTmU/edit#gid=0, which I updated just now with better color-coding (I didn't touch the order of elements). In a nutshell, the elements for all objects and collections should be in the order indicated on the spreadsheet, noting the following conditional: 1) if the Unit is RDC, then Note:Preferred Citation should be placed near the top. 2) If Unit is non-RDC, then Note:Preferred Citation should be located near the bottom. 3) Both Notes should have the label, "Cite This Work".

File: app/views/shared/fields/_show_raw.erb [hunk 1]
```diff
@@ -3,38 +3,60 @@
...
-<%= render :partial => 'shared/fields/relationships', :locals => component %>
-<%= render :partial => 'shared/fields/date', :locals => component %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'edition'}.merge(component) %>
-<%= render :partial => 'shared/fields/note', :locals => {:type => 'preferred citation', :custom_label => 'cite this work'}.merge(component) %>
-<%= render :partial => 'shared/fields/preferred_citation_note', :locals => component %>
+<%= render :partial => 'shared/fields/note', :locals => {:type => 'extent'}.merge(component) %>
+
+<% if unit_code == "rdcp" %>
+  <%= render :partial => 'shared/fields/note', :locals => {:type => 'preferred citation', :custom_label => 'cite this work'}.merge(component) %>
+  <%= render :partial => 'shared/fields/preferred_citation_note', :locals => component %>
```
[the remark this answers is not in the document]
Comment (VivianChu) [reply app/views/shared/fields/_show_raw.erb]: @lsitu - It seems like the field preferred_citation_note is old, not in used.  The spreadsheet doesn't mention it.  I can't find any record with that field - http://lib-hydratail-prod:8983/solr/blacklight/select?q=preferredCitationNote_json_tesim:*&wt=xml&debugQuery=true&rows=94  I can remove it if it's not in used.
Let me ask Ho Jung.

@hjsyoo - Do you know if there's any record that use this field dams:preferredCitationNote?
For example

    <dams:preferredCitationNote>
      <dams:PreferredCitationNote rdf:about="http://library.ucsd.edu/ark:/20775/bd3959888k">
        <rdf:value>Linked preferred citation note</rdf:value>
        <dams:displayLabel>Preferred citation</dams:displayLabel>
        <dams:type>preferred_citation</dams:type>
      </dams:PreferredCitationNote>
    </dams:preferredCitationNote>

**Comment by VivianChu** [comment]
@lsitu - You mentioned "the implementation may not reflect it".  I did use the conditional statement to check for the unit and place the Note:Preferred Citation to different place. 

https://github.com/ucsdlib/damspas/blob/262c157ba5d7dddc22290eb4a01d07ecc0684d77/app/views/shared/fields/_show_raw.erb#L12

https://github.com/ucsdlib/damspas/blob/262c157ba5d7dddc22290eb4a01d07ecc0684d77/app/views/shared/fields/_show_raw.erb#L102

**Comment by lsitu** [comment]
@hjsyoo @VivianChu Thanks for the clarification. I think it looks good and I can merge it. 👍
But the context of using `dams:preferredCitationNote` and `Note:preferred citation` that @VivianChu mentioned above is confusing.

**Comment by hjsyoo** [comment]
@arwenhutt Can you confirm that dams:preferredCitationNote is no longer used? I'm familiar with Note:preferred citation, but not dams:preferredCitationNote, if they are two different fields.

**Comment by lsitu** [comment]
@VivianChu I am moving forward to merge the PR now. I think we can create another ticket to cleanup the codes for `dams:preferredCitationNote` when needed.

**Comment by VivianChu** [comment]
Thanks @lsitu - Sounds good.

**Comment by hjsyoo** [comment]
Note from @arwenhutt : I think I can confirm that we don't use dams:preferredCitationNote currently.  ...There could be old data, but we are currently using the "Note:preferred citation"

**Pull request merged**